tiledb/dataframe_.py

```diff
@@ -176,25 +177,34 @@ def _get_column_infos(df, column_types, varlen_types):
     return column_infos
 
 
+def _get_schema_filters(filters):
+    if filters is True:
+        # default case, unspecified: use libtiledb defaults
+        return None
+    elif filters is None:
+        # empty filter list (schema uses zstd by default if unspecified)
+        return tiledb.FilterList()
+    elif isinstance(filters, (list, tiledb.FilterList)):
+        return tiledb.FilterList(filters)
+    elif isinstance(filters, tiledb.libtiledb.Filter):
+        return tiledb.FilterList([filters])
+    else:
+        raise ValueError("Unknown FilterList type!")
+
+
+def _get_attr_dim_filters(name, filters):
+    if isinstance(filters, dict):
+        # support passing a dict of filters per-attribute
+        return filters.get(name, None)
```

Comment:
I'd suggest `_get_schema_filters(filters.get(name, True))` to support all cases `_get_schema_filters` handles

Reply:
✔️